docker/services/api_compat/api_compat.finish

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+function log() {
+  echo $(date '+%Y-%m-%d %H:%M:%S %Z') "$@"
+}
+
+message="api compat terminated. "
+if [ $# -gt 0 ] && [ $1 -eq 0 ]; then
+  message+="Exit status 0 (consul triggered reload or service was stopped)"
+elif [ $# -gt 0 ] && [ $1 -eq 22 ]; then
+  message+="Exit status 22 (uwsgi failed to start app, check container logs)"
+elif [ $# -gt 0 ]; then
+  message+="Exit status $1"
```

Comment:
I'm also using status 10 in `run-lb-command` to indicate that there was a missing config key. We could also add this check ehre